### DISCUSSION TITLE
s_ba2 = kdm$fit$female$s_b2 in Step 3 is null

### DISCUSSION THREAD

**Discussion by wiwi456** [dayoonkwon/BioAge#41] [Q&A]
When projecting KDM bioage into the CALERIE using the kdm_calc function, 
 
![p1](https://github.com/user-attachments/assets/1b92759e-843f-4401-8040-348f78947274)

The fit is defined as the trained fit, while s_ba2= kdm$fit$female$s_b2 is a null value. Thus, s_ba2 was calculated separately in CALERIE using the following code. 
![p2](https://github.com/user-attachments/assets/f73c7093-d27d-4d6f-b6ba-d53514ab551d)
![p3](https://github.com/user-attachments/assets/04f18ae3-b91b-46d7-aae2-35ad10b7117a)

It confused me since I noticed that when calculate test modified KDM in NHANES4, the s_ba2 = train_fem$fit$s_ba2 is defined as the trained parameter.
 
![p4](https://github.com/user-attachments/assets/abe5a6e0-5d35-478b-aba0-dec5540d65be)

Could you please answer my confusion? Thank you in advance for your time!

**Comment by dayoonkwon** [comment]
This was a typo in the vignette example — `s_b2` should be `s_ba2`. Fixed in #56.
